fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -377,21 +367,37 @@ public void delete() {
                 }
             }
 
-            final Node parent;
+            final Node parent = getNode().getDepth() > 0 ? getNode().getParent() : null;
```

Comment:
This is likely a latent bug: adding the tombstone to the parent node (potentially a Pairtree) instead of the parent container.